loadJsonFile.py

```diff
@@ -0,0 +1,55 @@
+import googlemaps
+import json
+import requests
+
+API_KEY = 'AIzaSyDFtKtm0jnjckE-lB0LkO3SKlWmQIJush8';
```

Comment:
Same comment as in the other PR. API KEY shouldn't be checked in.

Reply:
OK! I know :) Thank you! I'll change it